solr/test-framework/src/java/org/apache/solr/cloud/SolrCloudTestCase.java

```diff
@@ -97,11 +97,9 @@ protected static SolrZkClient zkClient() {
     return cluster.getZkStateReader().getZkClient();
   }
 
-  /** if the system property is not specified, use a random value */
+  /** if the system property is not specified, default to false. The SystemProperty will be set in a beforeClass method. */
   public static boolean isPRS() {
-    return PRS_DEFAULT_PROP == null
-        ? random().nextBoolean()
-        : Boolean.parseBoolean(PRS_DEFAULT_PROP);
+    return Boolean.parseBoolean(System.getProperty(PRS_DEFAULT_PROP, "false"));
```

Comment:
Can EnvUtils be used here?

Reply:
Used it, in all places, but really in the tests, it will only be cared about in the `setPrsDefault()` since we overwrite the sysProp always.